xbmc/input/InertialScrollingHandler.cpp

```diff
@@ -172,9 +172,11 @@ bool CInertialScrollingHandler::ProcessInertialScroll(float frameTime)
       m_iLastGesturePoint.x += xMovement;
       m_iLastGesturePoint.y += yMovement;
 
-      //CLog::Log(LOGDEBUG, "@%f: %f,%f offset: %f, %f velocity: %f,%f dec: %f,%f", absoluteInertialTime,  m_iLastGesturePoint.x, m_iLastGesturePoint.y, xMovement, yMovement, m_iFlickVelocity.x, m_iFlickVelocity.y, m_inertialDeacceleration.x, m_inertialDeacceleration.y);
       //fire the pan action
-      g_application.OnAction(CAction(ACTION_GESTURE_PAN, 0, m_iLastGesturePoint.x, m_iLastGesturePoint.y, xMovement, yMovement, m_iFlickVelocity.x, m_iFlickVelocity.y));
+      if (!g_application.OnAction(CAction(ACTION_GESTURE_PAN, 0, m_iLastGesturePoint.x,
+                                          m_iLastGesturePoint.y, xMovement, yMovement,
+                                          m_iFlickVelocity.x, m_iFlickVelocity.y)))
+        m_bAborting = true; // we are done
```

Comment:
you don't have to bother, but I like `{}` to separate whitespace when multiple if lines are indented for a single if statement